### PR TITLE
BZ2149119: Cron job limitation in_MTC 1.7.6

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=2149119
MTC 1.7.6
OCP 4.12+
Preview: https://53871--docspreview.netlify.app/openshift-enterprise/latest/migration_toolkit_for_containers/mtc-release-notes.html#known-issues-1-7_mtc-release-notes

### CODE REVIEW

**Comment by sbeskin-redhat** [comment]
@pranavgaikwad Please review this PR.

**Review by stillalearner — APPROVED** [review]
lgtm

**Comment by RichardHoch** [comment]
LGTM

File: modules/migration-mtc-release-notes-1-7.adoc [hunk 1]
```diff
@@ -25,3 +25,4 @@ This release has the following known issues:
 * `MigPlan` custom resource does not display a warning when an AWS gp2 PVC has no available space. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1963927[*BZ#1963927*])
 * Direct and indirect data transfers do not work if the destination storage is a PV that is dynamically provisioned by the AWS Elastic File System (EFS). This is due to limitations of the AWS EFS Container Storage Interface (CSI) driver. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2085097[*BZ#2085097*])
 * Block storage for IBM Cloud must be in the same availability zone. See the link:https://cloud.ibm.com/docs/vpc?topic=vpc-block-storage-vpc-faq[IBM FAQ for block storage for virtual private cloud].
+* MTC 1.7.6 cannot migrate cron jobs from source clusters that support v1beta1 CronJob to OCP 4.12+ clusters, which do not support v1beta1 CronJob. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2149119[*BZ#2149119*])
```
Comment (adellape) [inline modules/migration-mtc-release-notes-1-7.adoc:28]:
```
OCP 4.12+
```
->
```
{product-title} 4.12 and later
```

Comment (adellape) [inline modules/migration-mtc-release-notes-1-7.adoc:28]: There's a mixture of "cron jobs" and "CronJob". You could change them all to lower case "cron job(s)" (no markup) as a general reference, or for the later ones that mention the API version you could do "v1beta1 `CronJob` objects". See general reference vs object reference in the [API object formatting](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting) repo guidelines.

You might also consider marking up `v1beta1`, as I see it often is elsewhere in the repo. E.g., "that support the `v1beta1` API version of `CronJob` objects".

Comment (sbeskin-redhat) [reply modules/migration-mtc-release-notes-1-7.adoc]: @adellape Thank you! Please take a look now.

**Comment by bergerhoffer** [comment]
The `branch/enterprise-4.13` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.12. And any PR going into `main` must also target the latest version branch (enterprise-4.13).

If the update in your PR does NOT apply to version 4.13 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Comment by sbeskin-redhat** [comment]
@bergerhoffer Can this PR be merged?

**Comment by bergerhoffer** [comment]
@sbeskin-redhat Can you add the 'merge-review-needed' label when you're ready for this to be merged? I can see that there are 4 commits though, so you'll need to squash at least before sending it. Thanks!

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.14` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.13. And any PR going into `main` must also target the latest version branch (enterprise-4.14).

If the update in your PR does NOT apply to version 4.14 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Comment by mjpytlak** [comment]
@sbeskin-redhat did you get approval from @pranavgaikwad outside of this PR? Looks like we are still waiting on it here.

**Review by pranavgaikwad — APPROVED** [review]

**Comment by sbeskin-redhat** [comment]
@mjpytlak 
He just approved it.

**Pull request merged**